src/authenticationinapp/authenticationinapplistener.cpp

```diff
@@ -236,7 +258,9 @@ void AuthenticationInAppListener::signIn(const QString& unblockCode) {
 
 void AuthenticationInAppListener::signInInternal(const QString& unblockCode) {
   NetworkRequest* request = NetworkRequest::createForFxaLogin(
-      m_task, m_emailAddressCaseFix, generateAuthPw(), unblockCode, m_urlQuery);
+      m_task, m_emailAddressCaseFix, generateAuthPw(), unblockCode,
+      m_fxaParams.m_clientId, m_fxaParams.m_deviceId, m_fxaParams.m_flowId,
+      m_fxaParams.m_flowBeginTime);
```

Comment:
I would have thought you needed, at least, scope. I was expecting you to need all of state, scope, accessType.

Just because the request works without these three doesn't mean it actually works:
* accessType, I thought, indicates we need a long-term refreshToken
* scopes are all the scopes we need for the support form, profile checking (including subscription checking), and in-app subscritpion to work.
* state is an important part of the integrity of the oauth flow (@jswinarton will have the details of what we need when on that bit)

Reply:
All is good here. See https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#post-accountcreate
The account creation doesn't need accessType, scopes, or state. Those are required only for the final step: `POST /oauth/authorization`